### DISCUSSION TITLE
리드잇진 글 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#50] [Q&A] [answered]
저번 오프라인 멘토링 당시에 리드잇진글을Activity에서 볼 수 있는 줄 알았는데.. 글 내용을 못찾고 있어서 질문드립니다.

**Comment by jongfeel** [comment, marked as answer]
아래 링크의 글입니다.

- https://github.com/jongfeel/Activity/issues/122
